.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -e ./general-superstaq[dev] -e ./qiskit-superstaq -e ./cirq-superstaq -e ./supermarq-benchmarks
+          pip install -e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq -e ./cirq-superstaq -e ./supermarq-benchmarks
```

Comment:
Is there a particular reason for the `pylint` to not install `[dev]` versions?

Reply:
nope, mostly there was just no reason to

Reply:
I guess no need to add it then?  Should I remove `[dev]` from some other places while we're at it?  Maybe it doesn't matter?

Reply:
it shouldn't really matter much i think - in practice the dev versions are often installed anyway because we usually use a shared cache. leaving it off can make that check a little faster after a cache miss, but the difference is probably pretty small